Remove VisualBasicProjectSystemPackage

davkean opened the pull request:
Fixes part of https://github.com/dotnet/project-system/issues/3817.

This package was never used and was solely present to find two strings. Delete package and move registration to Managed.VS.

davidwengier commented on src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Packaging/ProjectTypeRegistration.cs:
```diff
@@ -1,34 +1,17 @@
+// Visual Basic
 [assembly: ProjectTypeRegistration(
     projectTypeGuid: ProjectType.VisualBasic,
     displayName: "#1",                      // "Visual Basic"
```
Can we make these nicer names, given that we'll have a bunch of them in the same assembly now? Or do they have to be positional?

davkean replied:
We cannot, they must be integers.

davkean replied:
The design is based on Win32 resources and managed resources were kinda hacked on top of it.

davkean commented:
Note this slightly changed; fixed the name of the "strings" - they were wrong and should been without a '#' prefix. Left the resources in a VSPackage.resx that I moved over, as VS package resources need to be named this.
